docs/sources/flow/setup/start-agent.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Learn how to start, restart, and stop Grafana Agent Flow after it is installed
+title: Start, restart, and stop Grafana Agent Flow 
+menuTitle: Start
```

Comment:
"Start" might be misleading, since when I first saw it in the sidebar, I thought it meant "start" as in "start here." 

Should this be called something else, maybe "Run"?

Reply:
I've tweaked the nav text a little. I'm trying to align where possible with the core Grafana docs. So here it's now more parallel to the nav titles under: https://grafana.com/docs/grafana/latest/setup-grafana/ (Install Grafana Agent Flow, Configure Grafana Agent Flow, and Start Grafana Agent Flow.